README.md

```diff
@@ -1,4 +1,35 @@
 # Telegraf [![Circle CI](https://circleci.com/gh/influxdata/telegraf.svg?style=svg)](https://circleci.com/gh/influxdata/telegraf) [![Docker pulls](https://img.shields.io/docker/pulls/library/telegraf.svg)](https://hub.docker.com/_/telegraf/)
+[![Slack Status](https://img.shields.io/badge/slack-join_chat-white.svg?logo=slack&style=social)](https://www.influxdata.com/slack)
+# Logz.io Output Plugin
+
+This plugin sends metrics to Logz.io over HTTPs.
```

Comment:
I think that you do not want to change main `README.md` file. Something went wrong here...